Gems/RecastNavigation/Code/Source/Misc/RecastNavigationPhysXProviderConfig.cpp

```diff
@@ -16,7 +16,8 @@ namespace RecastNavigation
         {
             serializeContext->Class<RecastNavigationPhysXProviderConfig>()
                 ->Field("Use Editor Scene", &RecastNavigationPhysXProviderConfig::m_useEditorScene)
-                ->Version(1)
+                ->Field("Collision Group", &RecastNavigationPhysXProviderConfig::m_collisionGroupId)
+                ->Version(2)
```

Comment:
nit version is usually at the top of the serialize context so it's the first thing you see, especially if you plan to keep adding members

Reply:
Done.